release/128.0.yml

```diff
@@ -0,0 +1,590 @@
+release:
+  release_date: '2024-07-11'
+  text: |
+    **Thunderbird version 128.0 is only offered as direct download from
+    thunderbird.net and not as an upgrade from Thunderbird version 115 or
+    earlier. A future release will provide updates from earlier versions.**
+
+    **For more on all the new features in Thunderbird 128, see
+    [What’s New in Thunderbird 128](https://www.thunderbird.net/thunderbird/128.0/whatsnew/).**
+
+    **System Requirements:** [Details](/en-US/thunderbird/128.0/system-requirements/)
+
+    system_requirements: |
```

Comment:
Too much whitespace, it's lumped in with the text.
<img width="904" alt="image" src="https://github.com/thunderbird/thunderbird-notes/assets/97147377/00c36d04-5978-46d2-a613-aea1a0ba0475">